charts/discovery-api/values.yaml

```diff
@@ -2,6 +2,7 @@ global:
   auth:
     jwt_issuer: ""
     auth0_domain: ""
+    raptor_url: "http://raptor.admin/api/authorize"
   buckets:
```

Comment:
Where possible, the default should reflect the master chart's deployment. That means this should just be raptor since the master chart does it all in one namespace.

Reply:
Updated in deploy https://github.com/SmartColumbusOS/discovery_api-deploy/pull/188